### PR TITLE
HKG: Lower torque ramp limits for CAN-FD cars

### PR DESCRIPTION
**Pull request by sshane**

### CODE REVIEW

**Comment by sshane** [comment]
Changes in theoretical jerks:

```python
KIA EV6 2022 & HYUNDAI IONIQ 5 2022
============
Max lateral accel: 3.5 m/s^2
Old jerks: up: 3.89 m/s^3, down: 9.07 m/s^3
New jerks: up: 2.59 m/s^3, down: 2.59 m/s^3
```

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -29,6 +29,10 @@ def __init__(self, CP):
       self.STEER_DRIVER_ALLOWANCE = 250
       self.STEER_DRIVER_MULTIPLIER = 2
       self.STEER_THRESHOLD = 250
+      # These cars have significantly more torque than usual, limit torque rates
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:32]:
```suggestion
      # These cars have significantly more torque than usual, limit jerk
```

**Comment by HaraldSchafer** [comment]
LGTM

**Comment by sshane** [comment]
Greg will test this tonight

**Pull request merged**